psp/models/recent_history.py

```diff
@@ -377,6 +405,35 @@ def explain(self, x: X):
         explanation = self._regressor.explain(features, feature_names)
         return explanation
 
+    def _v7_get_capacity(self, dataset: xr.Dataset) -> float:
```

Comment:
I dont particualry like this names, but I understand why you called i this. 

Is there a better way to name this and in the function description say, we used this in v7?

Reply:
Sure will add a docstring. It's only there for backward compatibility and should not be used otherwise.

Reply:
ok, sounds good to do that